core/services/relay/evm/write_target.go

```diff
@@ -69,7 +81,192 @@ func NewWriteTarget(ctx context.Context, relayer *Relayer, chain legacyevm.Chain
 		return nil, err
 	}
 
-	return targets.NewWriteTarget(logger.Named(lggr, "WriteTarget"), id, cr, cw, config.ForwarderAddress().String(), gasLimitDefault), nil
+	pollPeriod, err := commonconfig.NewDuration(2 * time.Second)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create poll period: %w", err)
+	}
+
+	// TODO: Unsure what the timeout should be, I don't see an EVM config that corresponds to this. 
+	timeout, err := commonconfig.NewDuration(30 * time.Second)
```

Comment:
30s seems reasonable to me. Generalized Transmision state polling should be refactored in any case so we can make it configurable later.

Reply:
What do you mean it should be refactored to make it configurable? Isn't it [configurable already](https://github.com/smartcontractkit/chainlink-framework/blob/84df8e21833f0ad01bd82ce998c89d4ea048624e/capabilities/writetarget/write_target.go#L355)?